ignite/metrics/metric.py

```diff
@@ -548,9 +550,18 @@ def another_wrapper(self: Metric, *args: Any, **kwargs: Any) -> Callable:
             if len(attrs) > 0 and not self._is_reduced:
                 if ws > 1:
                     for attr in attrs:
+                        op_kwargs = {}
+                        if ":" in attr:
+                            idx = attr.index(":")
```

Comment:
Can't we do directly `attr, op = attr.split(":")` instead of using index etc ?

Reply:
If `:` is not existed it will raise an error 
```
ValueError: substring not found
```

Reply:
Yes that can be done after the check `if ":" in attr:`